vault/logical_system_helpers.go

```diff
@@ -106,40 +106,40 @@ var (
-			"replication/primary/demote":                                                           {logical.UpdateOperation},
-			"replication/primary/disable":                                                          {logical.UpdateOperation},
-			"replication/primary/secondary-token":                                                  {logical.UpdateOperation},
+			"replication/performance/primary/enable":                   {logical.UpdateOperation},
+			"replication/dr/primary/enable":                            {logical.UpdateOperation},
+			"replication/performance/primary/demote":                   {logical.UpdateOperation},
+			"replication/dr/primary/demote":                            {logical.UpdateOperation},
+			"replication/performance/primary/disable":                  {logical.UpdateOperation},
+			"replication/dr/primary/disable":                           {logical.UpdateOperation},
+			"replication/performance/primary/secondary-token":          {logical.UpdateOperation},
+			"replication/dr/primary/secondary-token":                   {logical.UpdateOperation},
+			"replication/performance/primary/revoke-secondary":         {logical.UpdateOperation},
+			"replication/dr/primary/revoke-secondary":                  {logical.UpdateOperation},
+			"replication/performance/secondary/generate-public-key":    {logical.UpdateOperation},
+			"replication/dr/secondary/generate-public-key":             {logical.UpdateOperation},
+			"replication/performance/secondary/enable":                 {logical.UpdateOperation},
+			"replication/dr/secondary/enable":                          {logical.UpdateOperation},
+			"replication/performance/secondary/promote":                {logical.UpdateOperation},
+			"replication/dr/secondary/promote":                         {logical.UpdateOperation},
+			"replication/performance/secondary/disable":                {logical.UpdateOperation},
+			"replication/dr/secondary/disable":                         {logical.UpdateOperation},
+			"replication/dr/secondary/operation-token/delete":          {logical.UpdateOperation},
+			"replication/performance/secondary/update-primary":         {logical.UpdateOperation},
+			"replication/dr/secondary/update-primary":                  {logical.UpdateOperation},
+			"replication/dr/secondary/license/status":                  {logical.ReadOperation},
+			"replication/dr/secondary/config/reload/(?P<subsystem>.+)": {logical.UpdateOperation},
+			"replication/recover":                                      {logical.UpdateOperation},
+			"replication/dr/secondary/recover":                         {logical.UpdateOperation},
+			"replication/dr/secondary/reindex":                         {logical.UpdateOperation},
+			"replication/reindex":                                      {logical.UpdateOperation},
+			//			"replication/status":                                                                   {logical.ReadOperation},
```

Comment:
Might be worthwhile to drop this or pull the comment from below up here and note that it will be handled separately below.

Reply:
I think dropping the comment is better. Without that comment the elements of the slice will all be aligned.